lib/id3c/cli/redcap.py

```diff
@@ -410,6 +417,28 @@ def is_complete(instrument: str, data: dict) -> bool:
     }
 
 
+def completion_status_field(instrument: str) -> str:
+    """
+    Returns the REDCap automatic field name for the completion status of
+    *instrument*.
+
+    If want to know the completion status itself, use :func:`is_complete`
+    instead.
+    """
+    # XXX TODO: It would be good to normalize *instrument* here, including:
+    #
+    #   - Lowercasing
+    #   - Replacing runs of whitespace and/or non-alphanumerics (?) with a
+    #     single underscore.
+    #   - Maybe: removing leading numbers?
+    #
+    # The full set of transformations REDCap applies aren't entirely clear to
+    # me at the moment, so I'm punting for now.  The caller must provide the
+    # internal names.
```

Comment:
That's fair. All I see in the redcap docs is the replacement of whitespace with underscore, but it would make sense to run some tests to see if there are other changes.

Reply:
Yeah, I did run some tests and enumerated my findings in the preceding lines, but I didn't want to invest more time once it became clear that the full behaviour isn't documented by REDCap.